modules/machine-node-custom-partition.adoc

```diff
@@ -0,0 +1,293 @@
+        ]
+      }
+    },
+    "version": "3.2.0"
+  }
+}
+
+        }
+      ]
+    },
+    "security": {
+      "tls": {
+        "certificateAuthorities": [
+          {
+            "source": "data:text/plain;charset=utf-8;base64,.....=="
+          }
+        ]
+      }
+    },
+    "version": "3.2.0"
+  },
+  "storage": {
+    "disks": [
+      {
+        "device": "/dev/nvme1n1", <1>
+        "partitions": [
+          {
+            "label": "var",
+            "sizeMiB": 50000 <2>
+            "startMiB": 0 <3>
+          }
+        ]
+      }
+    ],
+    "filesystems": [
+      {
+        "device": "/dev/disk/by-partlabel/var", <4>
+        "format": "xfs", <5>
+        "path": "/var" <6>
+      }
+    ]
+  },
+  "systemd": {
+    "units": [ <7>
+      {
+        "contents": "[Unit]\nBefore=local-fs.target\n[Mount]\nWhere=/var\nWhat=/dev/disk/by-partlabel/var\nOptions=defaults,pquota\n[Install]\nWantedBy=local-fs.target\n",
+        "enabled": true,
+        "name": "var.mount"
+      }
+    ]
+  }
+}
+----
+//Copied from installation-disk-partitioning-upi-templates.adoc
+<1> Specifies an absolute path to the AWS block device.
+<2> Specifies the size of the data partition in Mebibytes.
+<3> Specifies the start of the partition in Mebibytes. When adding a data partition to the boot disk, a minimum value of 25000 MB (Mebibytes) is recommended. The root file system is automatically resized to fill all available space up to the specified offset. If no value is specified, or if the specified value is smaller than the recommended minimum, the resulting root file system will be too small, and future reinstalls of {op-system} might overwrite the beginning of the data partition.
+<4> Specifies an absolute path to the `/var` partition.
+<5> Specifies the filesystem format.
+<6> Specifies the mount-point of the filesystem while Ignition is running relative to where the root filesystem will be mounted. This is not necessarily the same as where it should be mounted in the real root, but it is encouraged to make it the same.
+<7> Defines a systemd mount unit that mounts the `/dev/disk/by-partlabel/var` device to the `/var` partition.
+
+.. Extract the `disableTemplating` section from the `work-user-data` secret to a text file:
++
+[source,terminal]
+----
+$ oc get secret worker-user-data --template='{{index .data.disableTemplating | base64decode}}' | jq > disableTemplating.txt
+----
+
+.. Create the new user data secret file from the two text files. This user data secret passes the additional node partition information in the `userData.txt` file to the newly created node.
++
+[source,terminal]
+----
+$ oc create secret generic worker-user-data-x5 --from-file=userData=userData.txt --from-file=disableTemplating=disableTemplating.txt
+----
+
+. Create a new machine set for the new node: 
+
+.. Create a new machine set YAML file, similar to the following, which is configured for AWS. Add the required partitions and the newly-created user data secret:
```

Comment:
Similar to the following or using the following as a template?

Reply:
I would say _similar to the following_ as we have a note right after that suggests using an existing machine config as a template, so that you can get the correct labels and such for your cluster.